starfish/spots/_detector/blob.py

```diff
@@ -108,6 +108,9 @@ def image_to_spots(self, data_image: Union[np.ndarray, xr.DataArray]) -> SpotAtt
             self.overlap
         )
 
+        if fitted_blobs_array.shape[0] == 0:
+            return SpotAttributes.empty(extra_fields=['intensity', 'spot_id'])
```

Comment:
So pandas can't create a dataframe of size 0?

Reply:
I wasn't able to find a way to do it with a non-zero number of columns.